SWLOR.Game.Server/Data/Migrations/2020-09-29.2.sql

```diff
@@ -0,0 +1,212 @@
+﻿INSERT INTO
+  Perk (
+    ID,
+    Name,
+    IsActive,
+    BaseCastingTime,
+    Description,
+    PerkCategoryID,
+    CooldownCategoryID,
+    ExecutionTypeID,
+    IsTargetSelfOnly,
+    Enmity,
+    EnmityAdjustmentRuleID,
+    CastAnimationID,
+    ForceBalanceTypeID
+  )
+VALUES
+  (
+    88,
+    N'Speedy Crafting',
+    1,
+    0,
+    N'Reduces the amount of time it takes to craft items.',
+    4,
+    NULL,
+    0,
+    0,
+    0,
+    0,
+    NULL,
+    0
+  );
+INSERT INTO
+  PerkLevel (
+    ID,
+    PerkID,
+    Level,
+    Price,
+    Description,
+    SpecializationID
+  )
+VALUES
+  (
+    2319,
```

Comment:
I checked on this table and it isn't auto incrementing. Probably lost it when I shifted from SQL Server to MySQL. I'm making a note here because I need to make sure these IDs aren't in use on production. Also going to log a ticket so we can fix auto increments

Reply:
These are all good on production.